tools/ci_build/github/apple/test_ios_packages.py

```diff
@@ -56,6 +64,7 @@ def _test_ios_packages(args):
 
         # replace the target strings
         file_data = file_data.replace('${ORT_BASE_FRAMEWORK_ARCHIVE}', 'file:' + zip_file_path)
+        file_data = file_data.replace('${ORT_FRAMEWORK_NAME}', framework_name)
```

Comment:
>file_data [](http://example.com/codeflow?start=8&length=9)

could potentially specify the local pod directory in the Podfile
[https://guides.cocoapods.org/using/the-podfile.html#using-the-files-from-a-folder-local-to-the-machine](https://guides.cocoapods.org/using/the-podfile.html#using-the-files-from-a-folder-local-to-the-machine)
then perhaps we could just have one podspec

Reply:
Will do in a future PR